app/services/applications/accept.rb

```diff
@@ -77,11 +77,14 @@ def reject_other_applications_in_same_cohort!
     def other_accepted_applications_with_same_course_and_cohort
       return if application.blank?
 
+      not_withdrawn_training_statuses = Application.training_statuses.except(:withdrawn).keys.append(nil)
```

Comment:
Is there a reason not to use

```ruby
.where.not(training_status: nil)
```

alternatively can this be moved to a scope on the `Application` model so its available for use in other scenarios